test/framework/vmoperator/vmoperator.go

```diff
@@ -669,3 +688,82 @@ func ReconcileDependencies(ctx context.Context, c client.Client, dependenciesCon
 
 	return nil
 }
+
+// NOTE: code below is a fork of vm-operator's pkg/conditions (so we can avoid to import the entire project)
+
+const (
+	ReadyConditionType = "Ready"
+)
+
+type Getter interface {
+	client.Object
+
+	// GetConditions returns the list of conditions for a cluster API object.
```

Comment:
They literally have "conditions for a cluster API object." in their code?

Reply:
it is copy paste from vm-operator code, which I think is copy paste from CAPI